### ISSUE TITLE
Move ResetDB to Settings Activity

### ISSUE THREAD

**Issue by GoogleCodeExporter** [biotinker/LibreTasks#81]
```
The resetDB option should be moved to the Settings Activity.

```

Original issue reported on code.google.com by `case.and...@gmail.com` on 8 Jun 2010 at 11:01
- Blocked on: #80
- Merged into: #132
